test/integration/blocks-raw-handling.test.js

```diff
@@ -369,7 +387,7 @@ describe( 'Blocks raw handling', () => {
 			.join( '' );
 
 		expect( filtered ).toBe(
-			'<blockquote class="wp-block-quote"><p></p><cite>ribs</cite></blockquote>'
+			'<figure class="wp-block-quote"><blockquote></blockquote><figcaption><cite>ribs</cite></figcaption></figure>'
```

Comment:
related to my earlier comment, I believe this use of `<cite>` is in direct contradiction of how the spec indicates it should be used, so maybe we could remove the `<cite>` tags and leave it as `ribs`?

Reply:
First figs and now ribs? I'm getting hungry... 😋

And yes, I agree. The hardcoded inclusion of the `<cite>` is very much in contradiction with the spec, so it should be removed.